src/protobuf.rs

```diff
@@ -44,7 +44,9 @@ pub mod node {
         #[prost(message, tag = "9")]
         WindowFunc(::prost::alloc::boxed::Box<super::WindowFunc>),
         #[prost(message, tag = "10")]
-        WindowFuncRunCondition(::prost::alloc::boxed::Box<super::WindowFuncRunCondition>),
+        WindowFuncRunCondition(
+            ::prost::alloc::boxed::Box<super::WindowFuncRunCondition>,
+        ),
```

Comment:
Did we actually want to check in the formatting changes? (assuming this came from a `cargo fmt` as opposed to the original output?)

Reply:
The original PR included the formatting changes from `cargo fmt`. This PR undoes them in favor of the `cargo build` output